README.md

```diff
@@ -114,5 +133,5 @@ c:geo is distributed under [Apache License, Version 2.0](https://www.apache.org/
 - Twitter: https://twitter.com/android_gc
 - Facebook: https://www.facebook.com/android.geocaching
 - Google Play: https://play.google.com/store/apps/details?id=cgeo.geocaching
-- Live status: https://status.cgeo.org/
-- Developer chat: [#cgeo on freenode.net](https://webchat.freenode.net/?channels=%23cgeo)
+- Live status: https://www.cgeo.org/status.html
```

Comment:
@Lineflyer We should stay with `https://status.cgeo.org/`, isn't it?

Reply:
Currently it is the same, but who knows if the redirect changes in future. So better stay with status.cgeo.org for now.

Reply:
👍